front/components/assistant/conversation/input_bar/InputBarAttachments.tsx

```diff
@@ -71,14 +71,17 @@ export function InputBarAttachments({
 
         const isWebsiteOrFolder = isWebsite(dataSource) || isFolder(dataSource);
         const visual = isWebsiteOrFolder ? (
-          <Icon visual={logo} size="sm" />
+          <Icon visual={logo} />
```

Comment:
Is it intentional that we show a single icon for `isWebsiteOrFolder` in input bar but double icon for picker?

<img width="925" alt="Screenshot 2025-04-08 at 11 30 47" src="https://github.com/user-attachments/assets/295af985-ecfa-4367-9ad1-8f01ed0ca562" />

Reply:
Yes, otherwise we would have to display the same icon twice 😃